### PR TITLE
[W5.11][T11-A3]Anas Shakra

### PR DESCRIPTION
**Pull request by a-shakra**
Addressbook List command Lists entries in alphabetical order

### CODE REVIEW

**Comment by Sebry9** [comment]
good job!

is there no change in documentation though?

**Comment by a-shakra** [comment]
That's a very relevant point! List command entry in user guide has been altered to reflect this change.

File: src/seedu/addressbook/commands/ListCommand.java [hunk 1]
```diff
@@ -20,6 +24,20 @@
...
-        return new CommandResult(getMessageForPersonListShownSummary(allPersons), allPersons);
+        List<ReadOnlyPerson> listOfSortedPersons = new ArrayList<>();
+        for (int i = 0; i < allPersons.size(); i++) { // Copy List from unsorted array to array that will be sorted
+            listOfSortedPersons.add(allPersons.get(i));
+        }
+        
+        for (int i = 0; i < listOfSortedPersons.size()-1; i++) // Two for loops will determine the correct position to place an entry in
+            for (int j = 0; j < listOfSortedPersons.size()-i-1; j++)
+                if (listOfSortedPersons.get(j).getName().toString().toLowerCase().compareTo(listOfSortedPersons.get(j+1).getName()
+                        .toString().toLowerCase()) > 0) // the names are converted to string and to lowercase to take advantage of the compareTo function
```
Comment (jeffryhartanto) [inline src/seedu/addressbook/commands/ListCommand.java]: There are too many comments here, that usually means you think your code is not clear. Thus, you need to update the code, e.g. apply abstraction. Moreover, it's better to remove magic numbers.

**Comment by jeffryhartanto** [comment]
@a-shakra Some comments added. Please close the PR after reading comments.

IMPORTANT: If I remember correctly, you didn't pass the Week3 enhancement LO (will double-check my note on Monday). Thus, you need to implement 2 features for this LO. Moreover, for this feature you haven't updated the help command and test (either IO test or JUnit test). Therefore, you need to do two enchancement before the next tutorial and ensure that you update the test and documentation (UserGuide and Help).

**Pull request closed without merging**